docs/_docs/development/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: docs
+title: Maintaining Jekyll
+permalink: /docs/development/
```

Comment:
Permalinks of this section should all start with `/docs/maintaining/` if we go with @parkr proposal